docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -195,7 +195,7 @@ running on the node as follows
  * pods that tolerate the taint with a specified `tolerationSeconds` remain
    bound for the specified amount of time
 
-The above behavior is a beta feature. In addition, Kubernetes 1.6 has alpha
+In addition, Kubernetes 1.6 has alpha
 support for representing node problems. In other words, the node controller
```

Comment:
`In addition, Kubernetes 1.6 has alpha support for representing node problems.` @bsalamat does this support graduate to beta or GA after 1.6? If so, maybe we can also polish this wording?

Reply:
That part is still in alpha AFAIK.

Reply:
@bsalamat thanks for the information!
/lgtm